<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiWordEditor</class>
 <widget class="QDialog" name="UiWordEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>643</height>
   </rect>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../Resources/App.qrc">
    <normaloff>:/Menu/Menu/Edit.png</normaloff>:/Menu/Menu/Edit.png</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Term</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Value</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="0" rowspan="7" colspan="3">
    <widget class="QTextEdit" name="tedtTerm">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="tabStopWidth">
      <number>40</number>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="5" rowspan="7" colspan="2">
    <widget class="QTextEdit" name="tedtValueBrief">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="tabStopWidth">
      <number>40</number>
     </property>
     <property name="acceptRichText">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="3" colspan="2">
    <widget class="QToolButton" name="tbtnTranslate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Translate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QToolButton" name="tbtnGoogleTranslate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Google Translate</string>
     </property>
     <property name="text">
      <string>Google Translate</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources/App.qrc">
       <normaloff>:/Menu/Menu/GoogleTranslate.png</normaloff>:/Menu/Menu/GoogleTranslate.png</iconset>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QToolButton" name="tbtnDeeplTranslate">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="toolTip">
      <string>Deepl Translate</string>
     </property>
     <property name="text">
      <string>Google Translate</string>
     </property>
     <property name="icon">
      <iconset resource="../../../Resources/App.qrc">
       <normaloff>:/Menu/Menu/DeeplTranslate.svg</normaloff>:/Menu/Menu/DeeplTranslate.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QToolButton" name="tbtnTermCopy">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QToolButton" name="tbtnCheck">
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>136</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="5">
    <widget class="QToolButton" name="tbtnValueCopy">
     <property name="text">
      <string>Copy</string>
     </property>
    </widget>
   </item>
   <item row="8" column="6">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>242</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="0" colspan="7">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Detail</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="2" column="1">
        <widget class="QComboBox" name="cboTags"/>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="pbtnTagsEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/Menu/Menu/TagsEditor.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label">
         <property name="maximumSize">
          <size>
           <width>28</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>Tag</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QTextEdit" name="tedtValueDetail">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="tabStopWidth">
          <number>40</number>
         </property>
         <property name="acceptRichText">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="3">
        <widget class="QLabel" name="lblInfo">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Sunken</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="Web">
      <attribute name="title">
       <string>Web</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <widget class="QTextEdit" name="tedtValueWeb">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="tabStopWidth">
          <number>40</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Raw</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="tedtValueRaw">
         <property name="frameShape">
          <enum>QFrame::Box</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="tabStopWidth">
          <number>40</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QCheckBox" name="chkIsMarked">
     <property name="text">
      <string>Marked</string>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="2">
    <widget class="QCheckBox" name="chkIsLearned">
     <property name="text">
      <string>Learned</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="2">
    <widget class="QCheckBox" name="chkLowerCase">
     <property name="text">
      <string>Lower case</string>
     </property>
    </widget>
   </item>
   <item row="13" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="bbxButtons">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="4" column="3" colspan="2">
    <widget class="QToolButton" name="tbtnTermValueSwap">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Swap</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3" colspan="2">
    <widget class="QToolButton" name="tbtnTermPaste">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Paste</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3" colspan="2">
    <widget class="QToolButton" name="tbtnReset">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tedtValueBrief</tabstop>
 </tabstops>
 <resources>
  <include location="../../../Resources/App.qrc"/>
 </resources>
 <connections/>
</ui>
